install_config/aggregate_logging.adoc

```diff
@@ -416,24 +416,82 @@ The deployer creates an ephemeral deployment in which all of a pod's data is
 lost upon restart. For production usage, add a persistent storage volume to each
 Elasticsearch deployment configuration.
 
-The following example specifies a volume for an Elasticsearch replica (using a
-xref:../architecture/additional_concepts/storage.adoc#persistent-volume-claims[PersistentVolumeClaim]):
+The best-performing volumes are local disks, if it is possible to use
+them. There are some complications with doing so.
```

Comment:
The points that follow are the complications. Perhaps I should call them
something else... considerations?

On Thu, Aug 4, 2016 at 2:26 PM, Ashley Hardin <notifications@github.com>
wrote:

> In install_config/aggregate_logging.adoc
> <https://github.com/openshift/openshift-docs/pull/2599#discussion_r73576744>
> :
>
> > @@ -416,24 +416,82 @@ The deployer creates an ephemeral deployment in which all of a pod's data is
> >  lost upon restart. For production usage, add a persistent storage volume to each
> >  Elasticsearch deployment configuration.
> >
> > -The following example specifies a volume for an Elasticsearch replica (using a
> > -xref:../architecture/additional_concepts/storage.adoc#persistent-volume-claims[PersistentVolumeClaim]):
> > +The best-performing volumes are local disks, if it is possible to use
> > +them. There are some complications with doing so.
>
> what is an example complication?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/openshift-docs/pull/2599/files/1784a1c09d644badf0f598cea1d6528883d92537#r73576744>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AABz-gaBc2nMpYonf1MkT7KcC4gBFRSUks5qci7PgaJpZM4JbDXF>
> .
>